src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializerExt.cs

```diff
@@ -99,7 +99,7 @@ public static int WriteMetric(byte[] buffer, int cursor, Metric metric)
                     buffer[cursor++] = ASCII_LINEFEED;
                 }
             }
-            else
+            else if (metric.MetricType.IsHistogram())
```

Comment:
Ln44 says "if (!metric.MetricType.IsHistogram())", would this condition be mutually exclusive?

Reply:
It is today, but changed this to `metric.MetricType == MetricType.Histogram` just for clarity. See https://github.com/open-telemetry/opentelemetry-dotnet/pull/4337#discussion_r1149783660

Reply:
Nope, you're right, I buggered this. Fixed now and added a test.

Reply:
Closing as resolved.